Packs/Core/Playbooks/playbook-Uncommon_remote_scheduled_task_created.yml

```diff
@@ -0,0 +1,1433 @@
+                  simple: Core.OriginalAlert.event.actor_remote_ip
+                iscontext: true
+              right:
+                value:
+                  simple: '::'
+              ignorecase: true
+      ipRanges:
+        complex:
+          root: lists
+          accessor: PrivateIPs
+          transformers:
+          - operator: RegexExtractAll
+            args:
+              error_if_no_match: {}
+              ignore_case: {}
+              multi_line: {}
+              period_matches_newline: {}
+              regex:
+                value:
+                  simple: (\b(?:\d{1,3}\.){3}\d{1,3}\b/\d{1,2})
+              unpack_matches: {}
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 5ba5e082-b8f3-413f-89f6-40261ef6a811
+    type: title
+    task:
+      id: 5ba5e082-b8f3-413f-89f6-40261ef6a811
+      version: -1
+      name: Analyst Decision
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "42"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 2030
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "42":
+    id: "42"
+    taskid: 951c96f5-d9b9-4a75-802a-da1d01d66810
+    type: condition
+    task:
+      id: 951c96f5-d9b9-4a75-802a-da1d01d66810
+      version: -1
+      name: Analyst decision for suspicious cmd parameters
+      description: Based on suspicious command line parameters, analyst review is required to determine whether to take remediation actions such as disabling the scheduled task and blocking the IP if malicious, or to close the alert as a false positive.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      Disable Schedule Task:
+      - "3"
+      False Positive:
+      - "43"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 2160
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: |-
+          The following command-line contains a suspicious argument:
+
+          ${Core.OriginalAlert.event.scheduled_task_image_command_line}
+
+          Do you want to proceed and disable the scheduled task?
```

Comment:
missing a lot of context. 
like my comment above, please elaborate on the alert situation

Reply:
Done